content/content_service_test.go

```diff
@@ -546,9 +687,31 @@ func doesThingExist(uuid string, db neoutils.NeoConnection) (bool, error) {
 	return len(result) > 0, err
 }
 
+func checkRelationCount(db neoutils.NeoConnection, UUID string, assert *assert.Assertions) int {
+	countQuery := `
+		MATCH (t:Thing{uuid:{uuid}})-[r]->(x)
```

Comment:
No need for the variable setting since we are not using them:
```suggestion
		MATCH (:Thing{uuid:{uuid}})-[r]->()
```